internal/helm/client/client.go

```diff
@@ -160,3 +163,15 @@ func (c *ownerRefInjectingClient) Build(reader io.Reader, validate bool) (kube.R
 	}
 	return resourceList, nil
 }
+
+func containsResourcePolicyKeep(annotations map[string]string) bool {
+	if annotations != nil {
+		resourcePolicyType, ok := annotations[kube.ResourcePolicyAnno]
+		if ok {
+			resourcePolicyType = strings.ToLower(strings.TrimSpace(resourcePolicyType))
+			return resourcePolicyType == kube.KeepPolicy
+		}
+	}
+
+	return false
```

Comment:
Minor nit, but in order to minimize indentation, could you refactor to this?
```suggestion
if annotations == nil {
	return false
}
resourcePolicyType, ok := annotations[kube.ResourcePolicyAnno]
if !ok {
	return false
}
resourcePolicyType = strings.ToLower(strings.TrimSpace(resourcePolicyType))
return resourcePolicyType == kube.KeepPolicy
```

Reply:
Done as suggested. Locally tested again and it seems fine.